lib/modules/manager/ansible-galaxy/util.ts

```diff
@@ -5,7 +5,7 @@ export const blockLineRegEx = /^\s*((\w+):\s*(\S+))\s*$/;
 export const galaxyDepRegex = /[\w-]+\.[\w-]+/;
 export const dependencyRegex = /^dependencies:/;
 export const galaxyRegEx = regEx(
-  /^\s+(?<packageName>[\w.]+):\s*["'](?<version>.+)["']\s*/
+  /^\s+(?<packageName>[\w.]+):\s*["']?(?<version>.+)["']?\s*/
```

Comment:
```suggestion
  /^\s+(?<packageName>[\w.]+):\s*["']?(?<version>.+?)["']?\s*/
```
otherwise it can be too greedy

Reply:
Thanks. This regex is only used on single lines, which are extracted in advance. Only adding the suggested `?` would break the regex so added an '$' add the end of the regex